app/controllers/v1/workflow/steps_controller.rb

```diff
@@ -32,6 +32,16 @@ def uncomplete
     render json: V1::Workflow::StepSerializer.new(@step)
   end
 
+  def reorder
+    @process = Workflow::Instance::Process.find_by!(external_identifier: params[:process_id])
+    @step = @process.steps.find_by!(external_identifier: params[:id])
+    if Workflow::Instance::Process::ReorderSteps.run(@step, step_params[:after_position])
+      render json: V1::Workflow::StepSerializer.new(@step)
+    else
+      render status: 422
+    end
+  end
```

Comment:
This seems fine although I think the slightly more proper thing to do is return an errors object with validation errors.  I'm okay to skip for now.

Reply:
i can add that